### DISCUSSION TITLE
Can someone explain the 1s splash page?

### DISCUSSION THREAD

**Discussion by privacyguy123** [FastForwardTeam/FastForward#486] [Q&A] [answered]
### Body

I'm not understanding why the link can't be served directly, especially as you are just using a public API @ Apimon.de ...

### Version

0.0.0

### What browsers are you seeing the problem on?

Firefox

### What OS are you seeing the problem on?

Windows

**Comment by NotAProton** [comment, marked as answer]
we do not use Apimon for link1s, it's handled by client side JS

**Reply by privacyguy123** [reply]
Is there any need for the splash page yes or no? Why can't the addon serve the link directly?

EDIT: Ah there seems to be a slight misunderstanding - I am not talking about the website Link1s rather referencing the 1 second splash page that shows (even with 0 seconds set in Settings?) between clicking a link and "fast forwarding" it.
